packages/backend/src/controller/brevoController.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Body,
+  Controller,
+  // Get,
+  Post,
+  Route,
+  SuccessResponse,
+  TsoaResponse,
+  Res,
+  Example,
+  // Produces
+} from 'tsoa'
+import { createContact } from '../domain/features'
+import { BrevoRepository } from '../domain/spi'
+import { BrevoNotFoundError, BrevoBodyAttributes, BrevoResponse } from '../domain/types'
+import { requestBrevoAPI } from '../infrastructure/brevo-API'
+
+/**
+ * Defines how to access external services.
+ * Uses the "Repository" pattern, see README.md
+ */
+const rawlistIds: string[] = process.env['BREVO_LIST_IDS']?.split(',') || ['4']
+const listIds: number[] = rawlistIds.map(id => parseInt(id))
+
+const brevoRepository: BrevoRepository = {
+  postNewContact: async (email, attributes) => 
+    requestBrevoAPI(process.env['BREVO_API_TOKEN'] || '', email, listIds, attributes)
+}
+
+interface ErrorJSON {
+  message: string
+}
+
+interface BrevoNotFoundErrorJSON {
+  message: 'Contact not created'
+}
+
+interface ValidateErrorJSON {
+  message: 'Validation failed'
+  details: { [name: string]: unknown }
+}
+
+interface BrevoBody {
+  email: string,
+  attributes: BrevoBodyAttributes
+}
+
+const exampleBrevoResponse = {
+  id: 42
+}
+
+@SuccessResponse('200', 'OK')
+@Route('brevo')
+export class BrevoController extends Controller {
+  /**
+   * Add a new contact to TEE's Brevo service.
+   * Supply the email, the listId, and some attributes and receive the id of the contact created.
+   *
+   * @summary Adds a new contact to our Brevo list
+   *
+   * @example requestBody: {"email": "contact@multi.coop", "attributes": { "NOM": "Dupont", "PRENOM": "Camille", "TEL" : "0605040302", "SIRET": "83014132100034", "OPT_IN": true }}
+   */
+
+  @Example<BrevoResponse>(exampleBrevoResponse)
+  @Post('post_brevo_form')
```

Comment:
* Le verbe de l'API REST ne doit pas figurer dans son nom (car redondant). 
* Si on voulait mieux respecter l'esprit de l'architecture hexa, l'API (le controlleur) interne au projet (communication backend / frontend) ne devrait pas être spécifique à Brevo. 

Je suggère donc : 
```suggestion
  @Post('contact-information')
```

Dans la même idée, les types `BrevoBody`, `BrevoResponse` , `BrevoNotFoundError` etc. devraient avoir des noms plus agnostiques tel que `ContactInfoBody` et `ServiceNotFoundError`.